infrastructure/metadata/infrastructure/reftest/legacy/fuzzy-ref-2.html.ini

```diff
@@ -1,4 +0,0 @@
-[fuzzy-ref-2.html]
-  fuzzy:
-    if os == "mac" and product == "chrome": maxDifference=254-255;100-100
-    maxDifference=255;100-100
```

Comment:
Oops, I deleted a few rows that are actually used by default. I'll add them back.